style.css

```diff
@@ -721,3 +720,170 @@ li {
     line-height: 32px;
   }
 }
+
+.card-button-mobile.button {
+  display: flex;
+  justify-content: space-between;
+  background-color: #36b37e;
+  border-radius: 7px;
+  color: white;
+  padding: 12px 12px;
+  text-align: center;
+  font-size: 17px;
+  border: none;
+  height: 48px;
+  gap: 12px;
+}
+
+@media (min-width: 1050px) {
```

Comment:
- [ ] Well done styling the popup window so far. However, you may have to write some more CSS code to shape the widths and heights of it across multiple screen sizes. Please take a look at the two images down below.
- [ ] Please fix the styling of the `X` button
**Desktop**
![Screenshot (2)](https://user-images.githubusercontent.com/47625407/188766433-e4b9c2cb-3af4-41af-bbf2-a0268f67d969.png)
![Screenshot (1)](https://user-images.githubusercontent.com/47625407/188766442-874cc689-cc2e-47ad-92a0-7c204bbf817d.png)
**Mobile**
![Screenshot (3)](https://user-images.githubusercontent.com/47625407/188766471-c75ee666-5908-4fa2-bc3c-fe8d9f11a382.png)